frontend/packages/dev-console/src/components/import/git/GitSection.scss

```
@@ -0,0 +1,3 @@
+.odc-expand-collapse > div:first-child > button:first-child {
+  padding-left: 0px;
```

no need to add a unit with 0 value.
```suggestion
  padding-left: 0;
```